cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
```

Comment:
- Put weak keyword before it and change it to be optional OR put unowned keyword and assign an initial value "For making it weak reference and avoiding the memory leaks".